content/open-source/accoutrement.rst

```diff
@@ -0,0 +1,96 @@
+public: yes
```

Comment:
Suppose it's not technically important beyond consistent organization. We can change this back?